src/System.CommandLine.Tests/Parsing/CommandLineStringSplitterTests.cs

```diff
@@ -75,5 +75,25 @@ public void It_handles_multiple_options_with_quoted_arguments()
                          destination,
                          "--verbose");
         }
+
+        [Fact]
+        public void Internal_quotes_do_not_cause_string_to_be_split()
+        {
+            var commandLine = @"POST --raw='{""Id"":1,""Name"":""Alice""}'";
+
+            _splitter.Split(commandLine)
+                     .Should()
+                     .BeEquivalentTo("POST", "--raw='{Id:1,Name:Alice}'");
```

Comment:
This is wrong. The double quotes should be preserved.

Reply:
Minimal Changes have been done just to fix the mentioned issue and not to modify any existing logic.
It was preexisting logic to stripe away all the double-quotes. I tried to preserve it but it was conflicting with other test cases.